BackEnd/utils/image_moderation.py

```diff
@@ -24,3 +25,17 @@ def check_for_moderation(self):
             self.update_status()
             self.banner_logo["logo"] = self.profile.logo
         return self.moderation_is_needed
+
+    def approve_image(self):
+        if self.needs_moderation(self.profile.banner):
+            self.profile.banner.is_approved = True
+            self.profile.banner.save()
+            self.profile.banner_approved = self.profile.banner
+        if self.needs_moderation(self.profile.logo):
+            self.profile.logo.is_approved = True
+            self.profile.logo.save()
+            self.profile.logo_approved = self.profile.logo
+        self.profile.status = "approved"
```

Comment:
Use Enum

Reply:
Updated with choices of Profile status field